### ISSUE TITLE
querying monetdb doesn't have timeout

### ISSUE THREAD

**Issue by kbbgl** [kbbgl/e2ewd#2]
```[Thu Mar 07 22:11:40 UTC 2019] - [isMonetDBQuerySuccessful] Test successful for TCPW_2019: true```
but message just reached us:
```[Fri Mar 08 07:34:46 UTC 2019] - [isMonetDBQuerySuccessful] Test successful for Words Outsourced ProRata: false
[Fri Mar 08 07:34:57 UTC 2019] - [isMonetDBQuerySuccessful] ERROR could not connect to localhost:50005: No error
[Fri Mar 08 07:34:57 UTC 2019] - [isMonetDBQuerySuccessful] Test successful for DCP - Jan 19: false
[Fri Mar 08 07:35:08 UTC 2019] - [isMonetDBQuerySuccessful] ERROR could not connect to localhost:50036: No error
[Fri Mar 08 07:35:08 UTC 2019] - [isMonetDBQuerySuccessful] Test successful for TCPWDept_2019: false
[Fri Mar 08 07:35:18 UTC 2019] - [isMonetDBQuerySuccessful] ERROR could not connect to localhost:50033: No error
[Fri Mar 08 07:35:18 UTC 2019] - [isMonetDBQuerySuccessful] Test successful for TCPH_2019: false
[Fri Mar 08 07:35:18 UTC 2019] - [App.run] MonetDB test results: 
[Fri Mar 08 07:35:18 UTC 2019] - [App.run] {"TCPH":true,"TS Hours":true,"Work_Breakdown":true,"TCPWDept":true,"HR":true,"Helix Production":true,"Words Outsourced ProRata":false,"PE_2":true,"Productivity (Secure)":true,"Google_Finance_VAR":true,"TCPW_2019":true,"ISO_Training":true,"CARSv3":true,"External Spend":true,"TCPWCostCenter":true,"External Spend 2019":true,"Utilization (Secure)":true,"CARs":true,"Google":true,"WorkZone PROD":true,"TCPWDept_2019":false,"Words Received ProRata":true,"Words Received":true,"Utilization (Delivery)":true,"DCP - Jan 19":false,"360 Revenue - 17 + 18 FY":true,"Professional Services":true,"External Spend (New)":true,"Words Outsourced":true,"HeadCount":true,"Productivity (Delivery)":true,"TCPH_2019":false,"Helix Impact v2":true,"Quality Evaluation":true,"sisense_analytics":true,"Spend Report v3":true}
[Fri Mar 08 07:35:18 UTC 2019] - [SlackClient.sendMessage] Sent message to webhook. Response: ok
[Fri Mar 08 07:35:18 UTC 2019] - [ResultFile.write] Test succeeded: false```

**Comment by kbbgl** [comment]
Need to implement `public boolean waitFor(long timeout, TimeUnit unit)`

https://docs.oracle.com/javase/8/docs/api/java/lang/Process.html#waitFor-long-java.util.concurrent.TimeUnit-

**Comment by kbbgl** [comment]
Fixed in version 1.1.0.87124:

`if (process.waitFor(15, TimeUnit.SECONDS)){`
